packages/common/src/dto-types.ts

```diff
@@ -134,67 +148,46 @@ export class CreateStudentDto {
   concentration?: string;
 }
 
-export class UpdateStudentDto {
-  @IsOptional()
+export class OnboardStudentDto {
```

Comment:
A lot of changes here, but basically I make all the fields for a user REQUIRED for OnboardStudentDto as all fields are required if the student has finished onboarding

Reply:
can you use [Required<T>](https://www.typescriptlang.org/docs/handbook/utility-types.html#requiredtype)?

Reply:
Unless I misunderstood the docs, this isn't possible since these are classes and not types/interfaces. We require classes for class-validator.